docs/FOSS_puppet_server.md

```diff
@@ -1,62 +1,104 @@
 
 - [example42 PSICK Puppet Open Source Server automation](#example42-psick-puppet-open-source-server-automation)
     - [1. Vagrant installation](#1-vagrant-installation)
+        - [Prerequisites](#prerequisites)
+        - [Procedure](#procedure)
     - [2. Base OS installation](#2-base-os-installation)
+        - [Prerequisites](#prerequisites-1)
+        - [Procedure](#procedure-1)
 
 ## example42 PSICK Puppet Open Source Server automation
 
 This `control-repo` allows to spin up a `Puppet Open Source Server` in a fully automated way.
 
 One can use [vagrant](https://www.vagrantup.com/) or even use `PSICK` on a fresh base OS installation.
 
-First we need `puppet-agent` and `r10k` installed.
 
 ### 1. Vagrant installation
 
-For ```vagrant``` it is required to have a local `ruby` installation. Best option is to either use [rvm](https://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv). Then run:
+#### Prerequisites
+
+  - `Vagrant`
+  - local `ruby` installation. Best option is to use [rvm](https://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv)
+  - root rights
+  - `puppet-agent` and `r10k` packages installed
+
+
+#### Procedure
+
+Install either [rvm](https://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv). Then run:
+
 ```
     bundle install --path vendor/bundle
```

Comment:
unclear, more details are needed.

Reply:
Strictly speaking rvm or rbenv are not required, but it makes sense to mention them